apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/rubricSettingsModal.html.ts

```diff
@@ -465,6 +265,117 @@ export function RubricSettingsModal({ resLocals }: { resLocals: Record<string, a
   `;
 }
 
+function RubricItemRow(item: RubricData['rubric_items'][0] | null, index: number) {
+  const namePrefix = item ? `rubric_item[cur${item.id}]` : `rubric_item[new]`;
```

Comment:
Thanks for deduplicating all this markup!

I'm assuming there's already JavaScript in place to handle updating these IDs, the order, etc. after instantiating a new row from the template?

Reply:
Yes: https://github.com/PrairieLearn/PrairieLearn/blob/aa9182fd912451893e0f4195388e46b7960d4356/apps/prairielearn/assets/scripts/instructorAssessmentManualGradingInstanceQuestion.js#L496-L507